src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPageUserControlBase.vb

```diff
@@ -3337,7 +3341,19 @@ NextControl:
         ''' <returns></returns>
         ''' <remarks></remarks>
         Protected Function GetProjectPath() As String
-            Return CStr(GetCommonPropertyValueNative("FullPath")) 'CONSIDER: This won't work for all project types, e.g. ASP.NET when project path is a URL
```

Comment:
Hmm, actually maybe I'm wrong - it is calling through VSLangProj, sigh. I can't tell what object that is, can you set a break point and tell me what DTEProject.Object returns?